mirrord/layer/src/lib.rs

```diff
@@ -606,21 +606,18 @@ pub(crate) unsafe extern "C" fn close_detour(fd: c_int) -> c_int {
         .get()
         .expect("Should be set during initialization!");
 
-    if SOCKETS.lock().unwrap().remove(&fd).is_some() {
-        FN_CLOSE(fd)
-    } else if *enabled_file_ops
+    let res = FN_CLOSE(fd);
```

Comment:
Why do we want to call close also if the `fd` is remote and does not exist locally?

Reply:
Because each remote file is backed by a real fd locally and we don't want to leak it.

Reply:
Oh so until now we did, right?